.github/workflows/build-and-publish-rule-book.yaml

```
@@ -49,6 +49,11 @@ jobs:
             version: Версия
             warning: "Данная версия является неопубликованной сборкой из коммита в GitHub"
             sections: "sections/translated/ru"
+            luaengine: false
```

```suggestion
            luaengine: true
```
Don't change the settings for other languages.